lldb/source/Protocol/MCP/Protocol.cpp

```diff
@@ -81,6 +81,10 @@ bool operator==(const Request &a, const Request &b) {
   return a.id == b.id && a.method == b.method && a.params == b.params;
 }
 
+void PrintTo(const Request &req, std::ostream *os) {
+  *os << formatv("{0}", toJSON(req)).str();
+}
```

Comment:
If this is strictly for the unit tests, then should it live in a support file there?